tfhe/src/high_level_api/keys/inner.rs

```diff
@@ -2,39 +2,49 @@ use crate::core_crypto::commons::generators::DeterministicSeeder;
 use crate::core_crypto::prelude::ActivatedRandomGenerator;
 use crate::integer::public_key::CompactPublicKey;
 use crate::integer::CompressedCompactPublicKey;
+use crate::shortint::glwe_compression::{
+    CompressedGlweCompressionKey, CompressionParameters, GlweCompressionKey,
+    GlweCompressionPrivateKeys, GlweDecompressionKey, SeededGlweDecompressionKey,
```

Comment:
Decompression Key still has the Seeded naming

Reply:
Fixed